src/main/server.ts

```diff
@@ -44,7 +44,7 @@ new AppInsights().enable();
 new SessionStorage().enableFor(app);
 new CSRFToken().enableFor(app);
 new LanguageToggle().enableFor(app);
-new OidcMiddleware().enableFor(app);
+// new OidcMiddleware().enableFor(app);
```

Comment:
is there a reason we are commenting these out?

Reply:
My bad, didn't mean to commit this file, since that's only need to get the app working.